scripts/test_to_python.js

```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env node
```

Comment:
I'm still using this one for the new tests

Reply:
@pavelfeldman  Added it again